app/screens/settings/display_crt/display_crt.tsx

```diff
@@ -37,17 +38,22 @@ const DisplayCRT = ({componentId, currentUserId, isCRTEnabled}: Props) => {
 
     const close = () => popTopScreen(componentId);
 
-    const saveCRTPreference = useCallback(() => {
+    const saveCRTPreference = useCallback(async () => {
+        close();
```

Comment:
Is this something you may want to await for? Or is you just want it to be started before the rest of the function?